### ISSUE TITLE
Taskbar shown when child window closed when IgnoreTaskbarOnMaximize=true

### ISSUE THREAD

**Issue by gerthermans** [MahApps/MahApps.Metro#3108]
This zip file contains a simple c# project presenting the issue.
Steps to reproduce:
1. Start the project
2. Notice task bar is hidden (IgnoreTaskbarOnMaximize = true and WindowState=Maximized)
3. Click show window button to open child window
4. Close child window with close button
Expected result: Task bar is still hidden
Actual result: Taskbar is shown
[FullScreenMahAppsIssueCSharp.zip](https://github.com/MahApps/MahApps.Metro/files/1488004/FullScreenMahAppsIssueCSharp.zip)

### Environment

- MahApps.Metro _v1.5.0.23 (latest stable nuget package)
- Windows OS __10__
- Visual Studio __2015__
- .NET Framework __4.5.2__

**Comment by punker76** [comment]
@gerthermans Thx for the sample. Just tested it on my home pc and can't reproduce it with Win10 1709. Seems like the main bug is fixed on the OS itself. I get your issue with 1607 and 1703 too.

**Comment by gerthermans** [comment]
The issue still occurs with Win10 1709 when the child window is maximized and also has IgnoreTaskbarOnMaximize = true.
I've added an adjusted sample to show the issue.
Also in this sample I did add some code in comment in the closed eventhandler of childwindow.xaml.cs which can be used for a fix for now, but would be good if this issue could be fixed more generally in the next version.
[FullScreenMahAppsIssueCSharpChildAlsoIgnoreTaskbar.zip](https://github.com/MahApps/MahApps.Metro/files/1600362/FullScreenMahAppsIssueCSharpChildAlsoIgnoreTaskbar.zip)
